sql/sql_parse.cc

```diff
@@ -4294,7 +4294,7 @@ mysql_execute_command(THD *thd)
         effecting all columns in the table).
         SHOW CREATE VIEW require the SHOW_VIEW and SELECT ACLs on the table
         level.
-        NOTE: SHOW_VIEW ACL is checked when the view is created.
+        NOTE: SHOW_VIEW ACL is checked when the view is created for pre 5.0.1
```

Comment:
I don't understand this comment change. What does the version have to with this check? Can you please rephrase the whole "NOTE" to make it more clear what it's actually trying to explain?